packages/react/src/Bento/Bento.features.stories.tsx

```diff
@@ -0,0 +1,175 @@
+  <Bento>
+    <Bento.Item columnSpan={7} rowSpan={4} colorMode={ColorModesEnum.DARK} />
+    <Bento.Item columnSpan={5} rowSpan={4} colorMode={ColorModesEnum.LIGHT} />
+    <Bento.Item columnSpan={12} rowSpan={4} colorMode={ColorModesEnum.LIGHT} />
+    <Bento.Item columnSpan={12} rowSpan={4} colorMode={ColorModesEnum.DARK} />
+  </Bento>
+)
+
+export const ResponsiveItems: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+  </Bento>
+)
+
+export const ImageGallery: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item columnSpan={7} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={5} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+  </Bento>
+)
+
+export const Example: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item
+      columnSpan={{xsmall: 12, medium: 7}}
+      rowSpan={{xsmall: 3, small: 5}}
+      flow={{xsmall: 'row', small: 'row'}}
+      colorMode={ColorModesEnum.DARK}
+      visualAsBackground
+      className={styles['purple-background']}
+    >
+      <Bento.Content padding={{xsmall: 'normal', small: 'spacious'}} fixedBottomLink leadingVisual={<CopilotIcon />}>
+        <Bento.Heading as="h4" size="5">
+          Unlocking innovation at scale with AI-driven software development.
+        </Bento.Heading>
+        <Link href="#">Learn more about Copilot</Link>
```

Comment:
For this example, can we override the variant to achieve higher contrast please?

![Screenshot 2023-08-31 at 10 16 20](https://github.com/primer/brand/assets/13340707/f810a34f-44ab-496f-86ba-b537733ecb31)